ds-collector/ds-collector

```diff
@@ -591,22 +591,24 @@ collect_info_setup() {
   if [ "$is_dse" == "true" ]; then
     # Check DSE package install
     echo "DSE install: Checking install type..."
-    if [ -z "$dse_root_dir" ] && [ -d "/etc/dse" ] && [ -f "/etc/default/dse" ] && [ -d "/usr/share/dse/" ]; then
+    if [ -z "$dse_root_dir" ] && [ -d "/etc/dse" ] && [ -d "/etc/dse/cassandra" ] && [ -f "/etc/default/dse" ] && [ -d "/usr/share/dse/" ]; then
         echo "DSE install: package directories successfully found. Proceeding..."
         dse_root_dir="/etc/dse/"
         dse_bin_dir="/usr/bin/"
         dse_conf_dir="/etc/dse/"
+        configHome="/etc/dse/cassandra"
     elif [ -d "$dse_root_dir" ] && [ -d "$dse_root_dir/resources/cassandra/conf" ] && [ -d "$dse_root_dir/resources/dse/conf" ]; then
         echo "DSE install: tarball directories successfully found. Proceeding..."
         dse_bin_dir="$dse_root_dir/bin/"
         dse_conf_dir="$dse_root_dir/resources/dse/conf/"
+        configHome="$dse_root_dir/resources/cassandra/conf"
     else
         echo "DSE install: no package or tarball directories found, or no tarball directory specified."
         exit 1
```

Comment:
this `if … elif … exit 1 ; fi` approach means that there is no way (or purpose) to configure `dse_root_dir` or `dse_bin_dir` or `dse_conf_dir` or `configHome` variables in `collector.conf` 

should be permit conf file overrides? or should we warning that configured options are being ignore…?

Reply:
warning added.